lib/DDG/Spice/Pubmed/Search.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Pubmed::Search;
```

Comment:
When I remove the ::Search, to make this to DDG::Spice::Pubmed, it seems to be messing up the calls that use /js/spice/pubmed/abstract and /js/spice/pubmed/ids. Is it ok leaving this, or should I investigate why it isn't working as expected?